### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by sophiasph**
- [DEMO LINK](https://sophiasph.github.io/layout_catalog/)
- [TEST REPORT LINK](https://sophiasph.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/styles/cards.scss [hunk 1]
```diff
@@ -0,0 +1,100 @@
+@import "./stars";
+
```
Comment (Esceype) [inline src/styles/cards.scss:2]: Rename this file, if you card block inside so this file should call `card`.

File: src/styles/header.scss [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav {
```
Comment (Esceype) [inline src/styles/header.scss:14]: This is separate block so move it to separate file.

File: src/styles/stars.scss [hunk 3]
```diff
@@ -0,0 +1,21 @@
+.card__rating-stars {
```
Comment (Esceype) [inline src/styles/stars.scss:1]: It looks like card element and if you create it to use BEM mix methodology then you need to keep this class in card file but make it separate from stars block

File: src/index.html [hunk 4]
```diff
@@ -5,8 +5,407 @@
...
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="card" data-qa="card">
+        <img
+          class="card__product-image"
+          src="./images/imac.jpeg"
+          alt="Imac"
```
Comment (Esceype) [inline src/index.html:100]: Provide better naming for alt attribute

**Review by Esceype — CHANGES_REQUESTED** [review]
GJ!
Read the following comments

File: src/index.html (hunk 4 above)
Comment (sophiasph) [reply src/index.html:100]: why is it alt attribute name bad?

**Review by maxim2310 — CHANGES_REQUESTED** [review]
something went wrong with your demo link, plz fix it 
image

**Pull request closed without merging**